lib/resources/r4/related_person_patch.yaml

```diff
@@ -159,3 +160,26 @@ operations:
         <li>Requires a <a href="#test-name-id">test operation</a> to be provided for the `HumanName.id` of the name whose `family` is intended to be replaced.</li>
         <li>This operation is only supported for the first name in the list of related person names (<code>{index}</code> == 0).</li>
       </ul>
+
+  - name: replace-name-given
+    path: /name/{index}/given
+    operation: replace
+    type: String
+    url: https://hl7.org/fhir/R4/datatypes-definitions.html#HumanName.given
+    description: Replace the `HumanName.given` of the name at the given <code>{index}</code> in the list of related person names.
+    example: |
+      {
+        "path": "/name/0/given",
+        "op": "replace",
+        "value": [
+          "Harry",
+          "James"
+        ]
+      }
+    note: >
+      <ul>
+        <li>Requires a <a href="#test-name-id">test operation</a> to be provided for the `HumanName.id` of the name whose `given` is intended to be replaced.</li>
+        <li>This operation is only supported for the first name in the list of related person names (<code>{index}</code> == 0).</li>
+        <li>If there are fewer values provided than the HumanName.given list currently has, the additional existing values will be removed</li>
+        <li> Up to two `given` names may be provided. If additional given names are required they must be provided in the second value as a single string separated by blank spaces. </li>
```

Comment:
nit: Remove extra space at start of this line. I'm not sure if it will change anything visually, but good to be consistent.

Reply:
removed space https://github.com/cerner/fhir.cerner.com/commit/4c49f6dcee9374ca9c430f6fae3c07242de810ab#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dR184